lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc

```diff
@@ -98,11 +98,14 @@ function dosomething_signup_form_submit($form, &$form_state) {
   }
 
   if (variable_get('rogue_collection', FALSE)) {
+    $form_state['values']['source'] = 'phoenix-web';
```

Comment:
@df I deleted that line because I just noticed that the `source` was actually coming through to the Rogue database 
![screen shot 2017-06-07 at 4 27 45 pm](https://user-images.githubusercontent.com/9019452/26899887-8e30e006-4b9e-11e7-863c-73e24ef9b3b5.png)
- thanks for catching that!